project.yml

```diff
@@ -55,12 +55,14 @@ schemes:
         - BitwardenAutoFillExtension
         - BitwardenShareExtension
         - BitwardenShared
+        - AuthenticatorSyncShared
```

Comment:
🎨 Can this be alphabetized?

Reply:
I assume you mean the whole file? (i.e. it should be the first target as well). I'll have it updated to that in the next commit

Reply:
We're not the most consistent about alphabetization with the targets (notably, we put `BitwardenTests` after `Bitwarden` but before `BitwardenActionExtension`), but yeah, in general if we have a list of things that doesn't have an enforced order, by convention we like to alphabetize them.

Reply:
Updated in the latest commit. Let me know if we need to adjust anything else 👍